lib/reek/context_builder.rb

```diff
@@ -22,7 +23,7 @@ class ContextBuilder
     def initialize(syntax_tree)
       @exp = syntax_tree
       @element = Context::RootContext.new(exp)
-      @context_tree = process(exp)
+      @context_tree = build(exp)
```

Comment:
The naming here has been annoying me for sooooo long. "process" never really processed anything. It started to "build" the tree ;)